kaspresso/src/main/kotlin/com/kaspersky/kaspresso/testcases/core/TestRunner.kt

```diff
@@ -94,6 +103,28 @@ internal class TestRunner<InitData, Data>(
         resultException?.let { throw it }
     }
 
+    private fun injectKaspressoInKakao() {
```

Comment:
Maybe it makes sense to put this inject-specific code in a single place to avoid any confusion in the future.
Also, I see some code duplication here.